src/gluonts/torch/model/mqf2/estimator.py

```diff
@@ -0,0 +1,206 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import List, Optional, Dict, Any
+
+from gluonts.torch.model.deepar.estimator import DeepAREstimator
+from gluonts.torch.modules.loss import NegativeLogLikelihood, EnergyScore
+from gluonts.torch.distributions.mqf2 import MQF2DistributionOutput
+
+from . import MQF2MultiHorizonLightningModule, MQF2MultiHorizonModel
+
+from gluonts.core.component import validated
+from gluonts.time_feature import TimeFeature
+
+
+class MQF2MultiHorizonEstimator(DeepAREstimator):
```

Comment:
Why subclassing here?

Reply:
My original intention is to reduce duplication and have concise code.

Your suggestion certainly makes sense. We might just copy the code that we need from DeepAR.